test/test_tags.rb

```diff
@@ -554,6 +554,32 @@ def highlight_block_with_opts(options_string)
     end
   end
 
+  context "simple page with post linking containing special characters" do
+    setup do
+      content = <<CONTENT
+---
+title: Post linking
+---
+
+{% post_url 2016-11-26-special-chars-(+) %}
+CONTENT
+      create_post(content, {
+        "permalink"   => "pretty",
+        "source"      => source_dir,
+        "destination" => dest_dir,
+        "read_posts"  => true
+      })
+    end
+
+    should "not cause an error" do
+      refute_match(%r!markdown\-html\-error!, @result)
+    end
+
+    should "have the URL to the \"complex\" post from 2008-11-21" do
```

Comment:
:smile: I was actually asking Parker if that was allowed, but since you @Lunderberg implemented it anyways, can we switch the quotes to have it `"have the URL to the 'special-chars' post from 2016-11-26"` for homogeneity with other unit-tests.. also, there's more instances of `\""` inside **test names** within the same test-file.. can you patch that as well? Thanks